reaper/reaper.go

```diff
@@ -92,7 +93,15 @@ func (reaper reaper) reapPod(pod v1.Pod, reasons []string) {
 			"pod":     pod.Name,
 			"reasons": reasons,
 		}).Info("reaping pod")
-		err := reaper.clientSet.CoreV1().Pods(pod.Namespace).Delete(pod.Name, deleteOptions)
+		var err error
+		if reaper.options.evict {
```

Comment:
When I first saw notification of this pull request I thought it was a feature request. Was super cool to see it was a full PR and that it was done the same way popped into my head! :+1: